recipes/toad/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set name = "toad" %}
+{% set version = "0.0.58" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9dce218ac11eb9e3a80b233557694d0d349418a1ba517306919dff02595e5f07
+
+build:
+  number: 0
+  skip: True  # [py==27]
```

It makes little difference but I guess this better expresses the intent?
```suggestion
  skip: True  # [py2k]
```